src/webview/wkwebview/mod.rs

```diff
@@ -645,15 +645,8 @@ impl Drop for InnerWebView {
         }
       }
 
-      // WKWebview has a single WKProcessPool to manage web contents.
-      // The WKProcessPool is not reset even if WKWebview is deallocated.
-      // So we need to override the process by navigating to `about:blank`.
-      self.navigate("about:blank");
-
-      let _: Id<_> = Id::from_ptr(self.webview);
-      #[cfg(target_os = "macos")]
-      let _: Id<_> = Id::from_ptr(self.ns_window);
```

Comment:
I think ns_window is dropped in tao layer, so we don't need to drop it.

Reply:
We control its drop here iirc. Other `Rc<Window>`s are held by all hander/callbacks which we just free above. cc @lucasfernog

Reply:
tauri doesn't hold the rc IIRC btw, do we need to do something?

Reply:
No, I just think we still need to convert `self.ns_window` here IMHO.
`WebView` should own the `Window` and also release it when it's dropped.

Reply:
Just tested with removing this line and it seems fine. But I remembered we added these because of weird behavior in some combination when closing windows. Maybe take an eye on Tauri after this PR.